fbw-a32nx/src/systems/fmgc/src/guidance/lnav/LnavDriver.ts

```diff
@@ -463,7 +464,16 @@ export class LnavDriver implements GuidanceComponent {
   sequenceLeg(leg?: Leg, outboundTransition?: Transition): void {
     this.flightPlanService.active.sequence();
 
-    console.log(`[FMGC/Guidance] LNAV - sequencing leg. [new Index: ${this.flightPlanService.active.activeLegIndex}]`);
+    let secIndex = 1;
+    while (this.flightPlanService.hasSecondary(secIndex)) {
```

Comment:
Could consider using a `for` loop with some maximum iteration index here to make absolutely sure you don't run into an infinite loop.

Reply:
Done.